doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-rule-based-proc.adoc

```diff
@@ -0,0 +1,22 @@
+[id='test-scenarios-background-rule-based-proc']
+= Adding a background data in rule-based test scenarios
+
+Follow the procedure below to add and set a background data in rule-based test scenarios.
+
+.Prerequisites
+* The rule-based test scenario template are created for the selected test scenario. For more information about creating rule-based test scenarios, see xref:test-designer-create-test-template-rule-based-proc[].
+* The individual test scenarios are defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+
+.Procedure
+. Open the rule-based test scenarios in the test scenario designer.
+. Click the *Background* tab of the test scenarios designer.
+. Click an instance header cell in the *GIVEN* section to add a background data object field.
+. Select the data object from the *Test Tools* tab.
+. Click *Insert Data Object*.
+. Click a property header cell to add a background data object field.
+. Select the data object from the *Test Tools* tab.
+. Click *Insert Data Object*.
+. Optional: Click *Clear selection* to remove the selected data object.
+. To insert more properties of the data object, right-click the property header cell and select *Insert column right* or *Insert column left* as required.
```

Comment:
Minor suggestions - Optional change of sentences:

s/To insert more/To add more
s/of the data object/to the data object/in the data object